README.md

```diff
@@ -1,3 +1,25 @@
 # Simple Casper
 
 Layer-2 Casper daemon for POC4.
+
+## Setup
+### Requirements
+- `Python2.7`
+- `automake`
+- `libffi`
+- `tmux`
+
+### Getting started
+Make sure you have installed the requirements, and have created a virtual enviornment.
+Now install the project dependencies:
+```
+$ pip install -r requirements.txt
+$ python setup.py install
+```
+
+Next run the Casper simulator!
+```
+$ source ./tmux.sh
```

Comment:
@janx you're right this can just be `./tmux.sh` which is generally simpler--I'll use that instead